app/plugins/tag-debugger.js

```diff
@@ -4,7 +4,10 @@ export const commands = [
   'osmani',
 ]
 
+export const description = 'outline every DOM element with a random color, to visualize layout'
+
 export default async function() {
+  let i, A;
```

Comment:
otherwise console error in Chrome

Reply:
interesting.. thanks for finding it!

Reply:
yeah, i noticed the plugin wasn’t working until i added this line.  now that it's slightly more discoverable, it’ll work just in time